### PR TITLE
Modified the send confirmation mail functionality

### PR DESCRIPTION
**Pull request by Kirannegi02**
#1805

### Description:
Made the arrow tooltip available next to Send Confirmation Mail, whether checkbox is enabled or disabled, displayed a message as Mail not sent when checkbox is disabled below the Send Confirmation Mail. 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 1]
```diff
@@ -161,20 +161,21 @@
...
                         <span class="text-success font-weight-bold">
                             {{ __('Sent') }}
                         </span>
                     </label>
-                @endif
-            </div>
+                    @endif
+            </div> 
         </div>
+        <div v-if="show_on_select">Mail not sent</div>
```
Comment (GauravGusain98) [inline Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:178]: Why we need this?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php (hunk 1 above)
Comment (Kirannegi02) [reply Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:178]: As it was required that there should be message displayed as Mail not sent when checkbox is disabled below the Send Confirmation Mail. So, I have written it over here and put a condition (v-if="show_on_select") which will show" mail not sent" only when the checkbox is disabled.

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 2]
```diff
@@ -161,20 +161,21 @@
                 </div>
             </div>
             <div class="d-flex" v-if="status == 'paid'">
-                @if($invoice->payment_confirmation_mail_sent === 0)
+                    @if($invoice->payment_confirmation_mail_sent === 0)
                     <input type="checkbox" id="showEmail" class="ml-auto" @change="showEmail($event)" name="send_mail">
```
[the remark this answers is not in the document]
Comment (Kirannegi02) [reply Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:165]: If I am removing it, then the code is not working properly. The "mail not sent" text is being displayed in both cases whether the checkbox is enabled or disabled.

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 3]
```diff
@@ -227,9 +228,10 @@
 
         showEmail($event) {
             if (event.target.checked) {
-                this.show_on_select = true
-            } else {
                 this.show_on_select = false
+            } else {
+                this.show_on_select = true
+
```
[the remark this answers is not in the document]
Comment (Kirannegi02) [reply Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:234]: It is being used for "mail not sent" text.

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**